test/drbgtest.c

```diff
@@ -7,6 +7,9 @@
  * https://www.openssl.org/source/license.html
  */
 
+/* We need to use some deprecated APIs */
+#define OPENSSL_SUPPRESS_DEPRECATED
```

Comment:
Why can't we do the right thing here?

Reply:
We are testing the deprecated apis here as well - see below.

Reply:
This one can't change, as @t8m notes, it tests old stuff.